### PR TITLE
Add coin hub (BEP20)

### PR DESCRIPTION
**Pull request by bk3656**
Adding info for BEP20 token 'coin hub'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by bk3656** [comment]
hi we had the payment before done all procedure on case 22680 and as you said we can use that for this case as payment methods

**Comment by cryptomanz1** [comment]
#22680

**Comment by cryptomanz1** [comment]
• What is the purpose of this token? How will it benefit Trust Wallet users?
 • How is the token being distributed? Are you conducting an airdrop?
 • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
 • Is the token listed on a centralized/decentralized exchange?
 • Is a Smart Contract audit completed? Where can we find this information?

**Comment by bk3656** [comment]
hello
1-this is the utility token base on coin hub exchange platform the user who hold this token they can get discount on our platform and in the future we have to plan some great discount for trust wallet users on the fees in our platform
2-the token is not being with airdrop
3-yes we was and presale for 6 month in our dex platform 
4-yes the token is listed on centralized exchange (coinsibit.io with Usdt pair )and decentralized (pankekswap) also we are 
listed on coingecko.com website and all our detail is listed over there
5-no we didn't complete the audit

**Comment by Cryptocool1** [comment]
We require smart contract audit from reputable companies.

**Comment by Cryptocool1** [comment]
As per our policy, there is a minimum requirement of 10,000 holder addresses and 15,000 transactions, in order to filter out scam tokens.
See the criteria at : https://developer.trustwallet.com/assets/requirements .

**Comment by bk3656** [comment]
hi all
1-thanks for check our token on bscscan.com we already past 10k holder and 15 k transactions 
2-i'd like to hear your view wich company is  reputable companies for audit

**Review by bk3656 — COMMENTED** [review]
Coinhub listing request

File: blockchains/ethereum/assets/0xf68D4d917592f3a62417aCE42592F15296cc33A0/info.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "name": "Coin hub",
+    "description": "Utility token",
+    "symbol": "CHB",
+    "type": "ERC20",
+    "decimals": 8,
+    "website": "https://thecoinhub.io/",
+    "explorer": "https://etherscan.io/token/0xf68D4d917592f3a62417aCE42592F15296cc33A0",
+    "status": "active",
+    "id": "0xf68D4d917592f3a62417aCE42592F15296cc33A0",
+    "links": [
```
Comment (bk3656) [inline blockchains/ethereum/assets/0xf68D4d917592f3a62417aCE42592F15296cc33A0/info.json:11]: https://drive.google.com/file/d/1w-UbcVQ2s4uCEMobwCsksgF-RkEyXAY-/view?usp=share_link

**Review by bk3656 — COMMENTED** [review]
Please list and display Coinhub token CHB

**Pull request closed without merging**